### PR TITLE
Add examples for redfs run

### PR DESCRIPTION
**Pull request by d-hans**

### CODE REVIEW

File: README.fuse [hunk 1]
```diff
@@ -9,7 +9,7 @@ ninja
 cat << EOF | sudo tee /sbin/mount.fuse.passthrough_ll
 #!/bin/bash
 ulimit -n 1048576
-exec $(pwd)/example/passthrough_ll -ofsname="\$@"
+exec $(pwd)/example/passthrough_ll -ofsname="\$@" -o subtype=passthrough_ll
 EOF
```
Comment (bsbernd) [inline README.fuse:13]: Assuming `-o subtype=passthrough_ll` is ommited, does it still work?

File: common/rc [hunk 2]
```diff
@@ -4538,7 +4538,7 @@ init_rc()
 
 	# Sanity check that TEST partition is not mounted at another mount point
 	# or as another fs type
-	_check_mounted_on TEST_DEV $TEST_DEV TEST_DIR $TEST_DIR $FSTYP || exit 1
+	_check_mounted_on TEST_DEV $TEST_DEV TEST_DIR $TEST_DIR $FSTYP$FUSE_SUBTYP || exit 1
 	if [ -n "$SCRATCH_DEV" ]; then
```
Comment (bsbernd) [inline common/rc:4550]: Where is FUSE_SUBTYPE defined? In local.config? Also needs to go to the README then?

File: README.redfs.passthrough_ll [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Use the following local.config file:
+
+export TEST_DEV=non1
+export TEST_DIR=/mnt/test
+export SCRATCH_DEV=non2
+export SCRATCH_MNT=/mnt/scratch
+export FSTYP=redfs
+export FUSE_SUBTYP=.passthrough_ll
+export MOUNT_OPTIONS="-osource=/home/test/scratch,allow_other,default_permissions"
+export TEST_FS_MOUNT_OPTS="-osource=/home/test/test,allow_other,default_permissions"
```
Comment (bsbernd) [inline README.redfs.passthrough_ll:30]: I think it is better not to add an example for passthrough_ll. It just runs too slow, without additional passthrough_ll patches. Adding that example just causes confusion for anyone using it.

**Review by bsbernd — APPROVED** [review]

File: README.fuse (hunk 1 above)
Comment (d-hans) [reply README.fuse:13]: it would not work if next patch is there inside xfstests suite

File: common/rc (hunk 2 above)
Comment (d-hans) [reply common/rc:4550]: its already defined in local.config. I am not getting which README file you are talking about other than above one

File: README.redfs.passthrough_ll (hunk 3 above)
Comment (d-hans) [reply README.redfs.passthrough_ll:30]: i can add a comment about running redfs on it. Redfs should be run on passthrough_hp instead but its good to have examples of both for ourselves.

File: common/rc (hunk 2 above)
Comment (bsbernd) [reply common/rc:4550]: Ok, I had missed that

File: README.fuse (hunk 1 above)
Comment (bsbernd) [reply README.fuse:13]: Using -osubtype=${FUSE_SUBTYP} should work?

**Pull request merged**